Composer/plugins/runtimes/src/index.ts

```diff
@@ -0,0 +1,220 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+import path from 'path';
+import { promisify } from 'util';
+
+import rimraf from 'rimraf';
+import * as fs from 'fs-extra';
+
+import { copyDir } from './copyDir';
+import { IFileStorage } from './interface';
+
+const exec = promisify(require('child_process').exec);
+
+const removeDirAndFiles = promisify(rimraf);
+
+export default async (composer: any): Promise<void> => {
+  // register the bundled c# runtime used by the local publisher with the eject feature
+  composer.addRuntimeTemplate({
+    key: 'csharp-azurewebapp',
+    name: 'C#',
+    startCommand: 'dotnet run --project azurewebapp',
+    path: path.resolve(__dirname, '../../../../runtime/dotnet'),
+    build: async (runtimePath: string, _project: any) => {
+      composer.log(`BUILD THIS C# PROJECT! at ${runtimePath}...`);
+      composer.log('Run dotnet user-secrets init...');
+      // TODO: capture output of this and store it somewhere useful
+      const { initOut, initErr } = await exec('dotnet user-secrets init --project azurewebapp', {
```

Comment:
How does this work? The docs say that the return value of exec is `{ stdout, stderr }`.

Reply:
used { stderr: initErr } to map it now.